python/Ganga/Core/Sandbox/Sandbox.py

```diff
@@ -56,7 +56,15 @@ def createPackedInputSandbox(sandbox_files, inws, name):
 
 #
 # Future release with tarball module
-    with open(tgzfile, 'w') as this_tarfile:
+
+    if str(tgzfile).endswith('.gz') or  str(tgzfile).endswith('.tgz'):
+        file_format = 'gz'
+    elif  str(tgzfile).endswith('.bz2'):
+        file_format = 'bz2'
+    else:
+        file_format = ''
```

Comment:
`mimetypes.guess_type()` works only on the file name (it also works on remote URLs) so it would be usable. If it doesn't make the code any more readable or short then I'm happy to stick with the manual file extension checking.